src/components/touchables/TouchableNativeFeedback.tsx

```diff
@@ -1,22 +1,19 @@
-import React, { useEffect } from 'react';
 import { TouchableNativeFeedback as RNTouchableNativeFeedback } from 'react-native';
 import { tagMessage } from '../../utils';
 
 /**
  * @deprecated TouchableNativeFeedback will be removed in the future version of Gesture Handler.
  */
-const TouchableNativeFeedback: React.FC<
-  React.ComponentProps<typeof RNTouchableNativeFeedback>
-> = (props) => {
-  useEffect(() => {
+class TouchableNativeFeedback extends RNTouchableNativeFeedback {
+  componentDidMount() {
     console.warn(
       tagMessage(
-        'TouchableOpacity component will be removed in the future version of Gesture Handler.'
+        'TouchableNativeFeedback component will be removed in the future version of Gesture Handler.'
```

Comment:
```suggestion
```

Let's remove this `console.log` as in other `Touchables`

Reply:
done in [13b32eb](https://github.com/software-mansion/react-native-gesture-handler/pull/3294/commits/13b32eb0a9aa992e9fe72577ca8923157f73c9bc)